Formula/a/autobrr.rb

```diff
@@ -0,0 +1,45 @@
+class Autobrr < Formula
+  desc "Modern, easy to use download automation for torrents and usenet"
+  homepage "https://autobrr.com/"
+  url "https://github.com/autobrr/autobrr/archive/refs/tags/v1.36.0.tar.gz"
+  sha256 "631ee5f858c5952e1d7a8c27b13fe33aea9021ffd814bb540f0e494291168309"
+  license "GPL-2.0-or-later"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    # build the frontend
```

Comment:
```suggestion

```

Reply:
Just a nit. We normally only add comments to explain things that would otherwise not make as much sense without them. This is pretty self-explanatory.